src/lib/components/tablev2/SingleSelectableContent.tsx

```diff
@@ -96,6 +96,7 @@ export function SingleSelectableContent<
       <TableRow
         {...rowProps}
         isSelected={selectedId === row.id}
+        data-testid={selectedId === row.id ? 'row-selected' : undefined}
```

Comment:
Why is it necessary ?

Reply:
The previous test about checking for selected rows was checking props. Although it's possible (https://stackoverflow.com/questions/58623666/how-to-test-if-a-component-is-rendered-with-the-right-props-when-using-react-tes  - check out the subcomments of the "approved" answer), it's very roundabout approach, and it defeats the idea of the library - to test the DOM, not props. Selected row has uglified class name, so I needed something to use as selector

Reply:
I think the right way for selecting a row would be `getByRole('row')`

Reply:
Although we care if the row is selected and active, not how it's styled

Reply:
I think the right way would be to use `aria-selected` https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected

Reply:
Thanks, looks perfect of the use-case, wasn't aware of its existence